package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "visearch-javascript-sdk",
-  "version": "2.2.1-b2",
+  "version": "2.2.1-b3",
```

Comment:
if it is working, you can bump it to `2.2.2`
and update the read.me snippet section

Reply:
The working of the snippet is the same so there is no need to update the readme.